### DISCUSSION TITLE
One of my personal projects is not listed in Dashboard

### DISCUSSION THREAD

**Discussion by evelynstender** [vercel/community#6038] [Account Issue]
### Question

Hi Vercel team,

I created a project in the past for my personal website, but this project is not showing on my Vercel dashboard and I don't understand why. The website is still live, and the code is private on my GitHub. 

I remember having issues with login via email vs GitHub (they would lead to different dashboards, even though the email was the same on GitHub and Vercel). And I suspect that the solving of the login issue, caused my project (linked to my email) to disappear when I connected my GitHub to it?

Can someone help me figure out how to add my personal website back to my Vercel dashboard?

Thank you so much,
Evelyn

**Comment by amyegan** [comment]
Hi @evelynstender. It sounds l like you have two different Vercel accounts. How did you solve your login issue?
